baseplate/sidecars/live_data_watcher.py

```diff
@@ -135,10 +137,23 @@ def _load_from_s3(data: bytes) -> bytes:
         )
         raise LoaderException from e
 
-    s3_client = boto3.client(
-        "s3",
-        region_name=region_name,
-    )
+    if loader_config.get("anon") == True:
```

Comment:
**Note:** this requires an update to experiment-config to start sending over this new `anon` key. I did it this way since I didn't want to assume all services that use this codepath would want to be anonymizing the s3 client. The con here is that we're adding yet another key to check for from the ZK file.

However, if we think a) security is not a concern here or b) every service using this pathway would want an anonymized client anyways then I'm fine with leaving it as is.